### ISSUE TITLE
loss sometimes jumps at restart

### ISSUE THREAD

**Issue by travelguy95** [neuraloperator/neuraloperator#275]
Hello! I have been using your FNO with great interest and glee. Thank you again for this contribution.
I have one question: when I reload the FNO and continue training, occasionally I notice that the Loss jumps up at the beginning before declining. This doesn't happen with all datasets. 

I have been told that this may be due to random restart. Is there a way to prevent this? Thank you!!

FYI I am saving and loading my FNO as follows:
torch.save(model.state_dict(), PATH)
model = TheModelClass(*args, **kwargs)
model.load_state_dict(torch.load(PATH))
